wordpress-mobile/WordPress-iOS
# Notifications: Resizing Media Embeds on Rotation

### Testing Steps:
1. Launch WPiOS
2. Receive a Comment Notification with embedded images
3. Tap over the Notifications tab
4. Open the new notification
5. Verify that the assets should show up shortly, once available
6. Rotate the device
7. Verify that the assets get properly resized to fit the new width

### Notes:
This can be tested with the following images:

```
We'll do an inline image here <img src="https://i.cloudup.com/RiNQdiWYLe.gif">.

Another one here:

<img src="https://i.cloudup.com/RiNQdiWYLe.gif">

Another here:

<img src="https://cldup.com/XD_ZVbGXD7.png">

And last one: <img src="https://cldup.com/VSVBixYg33.png">
```

Needs Review: @aerych (thanks in advance Eric!)

Closes #2765
